.github/workflows/containers/github-action-ci/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM docker.io/library/ubuntu:22.04
+
+# swig is needed for lldb docs.
+
+# abi-dumper, autoconf, and pkg-config are required for the abi-dump workflow.
+# abi-compliance-checker is needed for the abi-compare workflow.
+# python3-setuptools and python3-psutil are required for the release-lit task.
+# Need to install curl for hendrikmuhs/ccache-action
+# Need nodejs for some of the GitHub actions.
+RUN apt-get update && \
+    apt-get install -y \
+    abi-compliance-checker \
```

Comment:
I'm assuming one of these packages pulls in a `gcc` based toolchain? It'll be needed for the documentation tests.

Reply:
It must, but it would be good to explicitly add it and probably clang too.